lib/smart_answer_flows/business-coronavirus-support-finder.rb

```diff
@@ -83,7 +97,7 @@ def define
         end
       end
 
-      # Q6
+      # Q7
```

Comment:
Minor: does these numbers actually help anyone? Since some questions may be conditional in general, I'm struggling to understand why these are here.

Reply:
No, just conventional for smart answers. I'm not a fan of them either just continued the convention. I'll nuke them.